### PR TITLE
refactor: new function name types

### PR DESCRIPTION
**Pull request by darzIdo**
complements https://github.com/taozhi8833998/node-sql-parser/pull/1827

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -65,7 +71,7 @@ export interface AggrFunc {
 }
 export interface Function {
   type: "function";
-  name: string;
+  name: { name: ValueExpr<string>[] };
```
Comment (taozhi8833998) [inline types.d.ts:74]: `name: { schema: string, name: ValueExpr<string>[] };`

**Review by taozhi8833998 — COMMENTED** [review]
function name can have `schema` key, please update it.

**Comment by taozhi8833998** [comment]
@darzIdo  I will update it

**Comment by taozhi8833998** [comment]
after adding the `schema`, will merge it.

**Comment by taozhi8833998** [comment]
after adding the `schema`, will merge it.

**Comment by darzIdo** [comment]
@taozhi8833998 made the change - i just set the `schema` to be optional, as I see it's not always provided, right?

**Comment by taozhi8833998** [comment]
> @taozhi8833998 made the change - i just set the `schema` to be optional, as I see it's not always provided, right?

yes, you are right

**Pull request merged**